examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml

```diff
@@ -22,11 +22,11 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>io.helidon.examples.quickstarts</groupId>
     <artifactId>helidon-standalone-quickstart-mp</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.0.0-M3</version>
     <name>Helidon Standalone Quickstart MP Example</name>
 
     <properties>
-        <helidon.version>2.0.0-SNAPSHOT</helidon.version>
+        <helidon.version>2.0.0-M3</helidon.version>
```

Comment:
Same as above - this should reference snapshot

Reply:
Sure. I cannot build the project with "-SNAPSHOT"

Reply:
Unresolving, this must reference the SNAPSHOT version. You need to build the project in order to use SNAPSHOT versions.

Reply:
> Unresolving, this must reference the SNAPSHOT version. You need to build the whole project in order to use SNAPSHOT versions.

Updated the version to "2.0.0-SNAPSHOT" https://github.com/oracle/helidon/pull/1849/commits/89cd9c4a681d5d9a48d3e0ab07593d4a5154d774 (but this maybe an oracle internal only available version?)

Reply:
It's not an internal version, this is the latest version that you have to build from source.

Reply:
Was able to successfully build the whole project using `2.0.0-SNAPSHOT`